.changeset/wise-lemons-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/fsm': patch
+---
+
+Passed state type generic to the Config interface
```

Comment:
Changesets appear later in changelogs and this sounds really vague to a potential reader, what about something like this:
```suggestion
`StateMachine.Config` type accepts now a third type parameter - `TState` - similarly to other existing types. When provided it provides helpful intellisense when defining the state chart transitions.
```